packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -0,0 +1,30 @@
+/* global before, browser, Terra */
+
+describe('Overlay', () => {
+  describe('Light Themed', () => {
+    before(() => browser.url('/#/tests/overlay-tests/light-themed'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+    Terra.should.themeEachCustomProperty({
+      '--terra-overlay-content-color': 'red',
+      '--terra-overlay-light-background-color': 'green',
+      '--terra-overlay-light-image': 'linear-gradient(red, green)',
+      '--terra-overlay-light-opacity': '0.8',
+      '--terra-overlay-content-padding': '1000px',
```

Comment:
Does the content padding need to be tested for both light and dark themes?

Reply:
The content theme variables is not specific to light or dark theme its applied across all the themes.